.appveyor.yml

```diff
@@ -8,9 +8,8 @@ environment:
   GOPATH: C:\gopath
   PATH: C:\go\bin;C:\tools\mingw64\bin;$(PATH)
   CGO_ENABLED: 1
-  GO111MODULE: off
   matrix:
-    - GO_VERSION: 1.13.12
+    - GO_VERSION: 1.14.4
```

Comment:
Not sure if it's safe to use Go 1.14 (currently, I think the plan is to skip 1.14, and go to 1.15; https://github.com/containerd/containerd/pull/4050)

Reply:
yeah I'm thinking maybe use go 1.14 for module vendoring like I see over on runc? That or we need to do some EINTR validation..

Reply:
Do we know why they use go 1.14 for vendoring but 1.13 for everything else?